src/sentry/integrations/slack/webhooks/base.py

```diff
@@ -129,16 +132,76 @@ class SlackCommandDispatcher(MessagingIntegrationCommandDispatcher[Response]):
     endpoint: SlackDMEndpoint
     request: SlackDMRequest
 
+    # Define mapping of messages to halt reasons
+    @property
+    def TEAM_HALT_MAPPINGS(self) -> dict[str, MessageCommandHaltReason]:
```

Comment:
built this by taking a look at the control flows possible from the `team_link` and `team_unlink` methods:

https://github.com/getsentry/sentry/blob/9e0e08145e8074f7c4cc3af4a1b493d7a5dad0db/src/sentry/integrations/slack/webhooks/command.py#L85
https://github.com/getsentry/sentry/blob/9e0e08145e8074f7c4cc3af4a1b493d7a5dad0db/src/sentry/integrations/slack/webhooks/command.py#L133